src/lib/utils.ts

```diff
@@ -24,4 +26,14 @@ function firstPromiseFulfilled<T>(ps: Promise<T>[]) {
   return invertPromise(Promise.all(ps.map(invertPromise)));
 }
 
-export { randomString, firstPromiseFulfilled };
+function protobufToString(message: Uint8Array) {
+  return util.base64.encode(message, 0, message.length);
```

Comment:
I would not call this `util`. Just do `import * as protobufjs from 'protobufjs';` or `import protobufjs from 'protobufjs';`. Then it's clearer. The `util` here is strange given that this entire module is called `utils`.

Also return type? String to buffer and buffer to string? Are these only for debugging?

Reply:
easy, done :+1: